code/LINDA/LINDA_turf_tile.dm

```diff
@@ -288,10 +288,14 @@
 
 
 /atom/movable/var/pressure_resistance = 5
+/atom/movable/var/last_forced_movement = 0

```

Comment:
Defining variables is basically the only time we use relative pathing
And even then we could get far more relative than we are now